omnibus/.kitchen.yml

```diff
@@ -63,6 +68,13 @@ platforms:
   # By adding an `i386` to the name the Omnibus cookbook's `load-omnibus-toolchain.bat`
   # will load the 32-bit version of the MinGW toolchain.
   - name: windows-2012r2-standard-i386
+    provisioner:
+      attributes:
+        omnibus:
+          build_user:          jenkins
+          build_user_group:    Administrators
+          build_user_password: jenkins
```

Comment:
We probably want to switch this back to `vagrant` stuff before merge correct? That will ensure people don't need to do a logout/login dance to do a build in their VM.